MapleServer2/Packets/MeretMarketPacket.cs

```diff
@@ -118,17 +118,17 @@ public static Packet Promos(List<MeretMarketItem> items)
                 }
             }
 
-            pWriter.WriteByte(0);
-            pWriter.WriteByte(0);
-            pWriter.WriteByte(0);
-            pWriter.WriteByte(0);
-            pWriter.WriteByte(0);
-            pWriter.WriteByte(0);
-            pWriter.WriteByte(0);
-            pWriter.WriteByte(0);
-            pWriter.WriteByte(0);
-            pWriter.WriteByte(0);
-            pWriter.WriteByte(0);
+            pWriter.WriteByte();
+            pWriter.WriteByte();
+            pWriter.WriteByte();
+            pWriter.WriteByte();
+            pWriter.WriteByte();
+            pWriter.WriteByte();
+            pWriter.WriteByte();
+            pWriter.WriteByte();
+            pWriter.WriteByte();
+            pWriter.WriteByte();
+            pWriter.WriteByte();
```

Comment:
Maybe use `pWriter.WriteZero(10)`?

Reply:
I'm opposed to this because it doesn't indicate the packet structure. These single bytes might mean something we don't know yet.